packages/components/psammead-figure/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
```

Comment:
I think this should have a storybook entry

Reply:
@ChrisBAshton and I discussed this and decided against it. There's not to show at the moment, but I agree something is needed down the line.

Reply:
Just had another chat about this - my bad, we should have one. It's just that we didn't have one in Simorgh and it felt odd that you had to add it.

But we do want an entry for each one, as Storybook is going to be the way to view what components are available and read their READMEs, etc - so even if we can't really _see_ it, it's worth having.

Reply:
Ok 👍

Reply:
Just adding a block label on this PR as I will wait until https://github.com/BBC-News/psammead/pull/225 (Create initial psammead-image) is merged.

When that is complete I will add the story for Figure.